tests/m2m_and_m2o/tests.py

```diff
@@ -5,6 +5,13 @@
 
 
 class RelatedObjectTests(TestCase):
+
+    def test_related_objects_have_name_attribute(self):
+        for Model in (Issue, User, UnicodeReferenceModel,):
```

Comment:
It would be better to structure this test to pull out the expected fields and assert each of them individually. That way if the test fails it'll be easier to debug, and also you ensure that some assertions are actually made (some changes to the model might make it so that to fields pass `is_reverse_object` and then the test does nothing).